testsuite/MDAnalysisTests/analysis/test_nuclinfo.py

```diff
@@ -174,3 +174,13 @@ def test_hydroxyl(u, seg, i, expected_value):
 def test_pseudo_dihe_baseflip(u, bp1, bp2, i, seg1, seg2, seg3, expected_value):
     val = nuclinfo.pseudo_dihe_baseflip(u, bp1, bp2, i, seg1, seg2, seg3)
     assert_almost_equal(val, expected_value, decimal=3)
+
+
+@pytest.mark.parametrize('b1, b2, seg1, seg2, expected_value', (
+        (2, 3, 'RNAA', 'RNAA', 15.490582),
+        (10, 11, 'RNAA', 'RNAA', 43.71941),
+        (7, 17, 'RNAA', 'RNAA', 168.37481),
+))
+def test_angle_between_base_planes(u, b1, b2, seg1, seg2, expected_value):
+    val = nuclinfo.angle_between_base_planes(u, b1, b2, seg1, seg2)
+    assert_almost_equal(val, expected_value, decimal=3)
```

Comment:
maybe `assert_allclose()` for new code: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html

Reply:
Okay. Would you like me to move to `assert_allclose()` for all the tests in nuclinfo? 

Although I think it is reasonable to stick with `assert_almost_equal()` since we are not looking for precision beyond 3 decimals.

Reply:
No, just here for now; I can't think of a reason to use the discouraged approach in new code.

Reply:
Updated. Thanks.